models/pathology_nuclei_segmentation_classification/docs/README.md

```diff
@@ -86,8 +84,9 @@ For more details usage instructions, visit the [MONAI Bundle Configuration Page]
 
 - Run first stage
 ```
-python -m monai.bundle run --config_file configs/train.json --network_def#pretrained_url `PRETRAIN_MODEL_URL` --stage 0
+python -m monai.bundle run --config_file configs/train.json --network_def#pretrained_url $PRETRAIN_MODEL_URL --stage 0
```

Comment:
Why add "$" here?

Thanks.

Reply:
Oh, you exported it as a system env var?

Reply:
No, just because the bash shell usually uses this"$" for the variable.

Reply:
I think here no need to mention PRETRAIN_MODEL_URL.
why not use the same way as in:
https://github.com/Project-MONAI/model-zoo/pull/370/files#diff-a739ca80d65b7994071d4c6f0fc9a2397f698b958b98f1d314056f1fe260d2acR26

Reply:
I'm fine with the updates. Hi @Nic-Ma , do you have other comments here?

Reply:
Hi @Nic-Ma and @yiheng-wang-nv, have removed `PRETRAIN_MODEL_URL` in this commit https://github.com/Project-MONAI/model-zoo/pull/370/commits/f1a93f8263ece4400f8a881ddc8a29d68d26ea8f.
And also explained that if users want to use the pre-trained weights they can specify `--network_def#pretrained_url your-pretrain-weights-url` in the training command.
Thanks!

Reply:
Hi @KumoLiu ,

Please use "<your pretrain weights URL>" instead, that's common practice for user-specified path.

Thanks.

Reply:
Thanks, @Nic-Ma, have updated it in this [commit](https://github.com/Project-MONAI/model-zoo/pull/370/commits/2cbf24044abbe7e6953eac51b8a2904a378558a8).